### DISCUSSION TITLE
Add "Delete entries from this site" action to element index

### DISCUSSION THREAD

**Discussion by Mosnar** [craftcms/cms#9022] [Ideas] [answered]
### Description

There's currently no way to bulk-delete entries from only the current site when the section's propagation mode is set to "Let each entry choose which site it is saved to" - the only way to do it is to go into each individual entry and select "Delete entry from this site"

This is a huge pain since when you switch a propagation mode to this, all of the entries that were previously just disabled in the specific site now need to be manually deleted one-by-one.

**Comment by brandonkelly** [comment, marked as answer]
This has been added for 3.7! ✨ #9119

**Reply by brandonkelly** [reply]
3.7.0-beta.6 is out now with this action.
